tests/produce_integration_test_data.py

```diff
@@ -357,12 +312,28 @@ def produce_lc_power_spectra(redshift, **kwargs):
                 "kinetic_temp_neutral",
             }
         ]
-    if not astro_options.USE_MINI_HALOS:
+    if not inputs.astro_options.USE_MINI_HALOS:
         quantities.remove("J_21_LW")
+    if not inputs.astro_options.INHOMO_RECO:
+        quantities.remove("cumulative_recombinations")
+
+    return quantities
 
+
+def produce_lc_power_spectra(redshift, **kwargs):
+    options = get_all_options_struct(redshift, lc=True, **kwargs)
+    print("----- OPTIONS USED -----")
+    print(options)
+    print("------------------------")
+
+    # NOTE: this is here only so that we get the same answer as previous versions,
+    #       which have a bug where the max_redshift gets set higher than it needs to be.
+    node_z = options["inputs"].node_redshifts
```

Comment:
Which previous version are we trying to match here? We should just set the standard in this PR.

Reply:
I fixed the behaviour but forgot to delete the comment, it's done now